specification/sbi_atee.adoc

```diff
@@ -1478,15 +1478,19 @@ one or more page mappings to the G-stage translation structure for a TVM.
 The pages to be used for the G-stage page table structures must have been converted
 (and tracked) by the TSM as TEE pages; otherwise this operation will not succeed.                                                                     
 
-| <<sbi_tee_host_add_tvm_measured_pages, sbi_tee_host_add_tvm_measured_pages>> | Add 
-a page for an existing mapping for a TVM page - this add_pre must be performed before 
-finalization of the TVM measurement via sbi_tee_host_finalize_tvm. For this 
-operation, the VMM must provide the page contents that get copied into 
-confidential memory pages for the TVM (and get tracked, encrypted etc). The 
-contents of these pages are also measured via the sbi_tee_host_add_tvm_measured_pages, 
-including the GPA at which the page is mapped. After the TVM msmt is 
-finalized via sbi_tee_host_finalize_tvm, no more pre-add pages are allowed by 
-the TSM for that TVM.
+| <<sbi_tee_host_add_tvm_measured_pages, sbi_tee_host_add_tvm_measured_pages>> | Copies the
+given number of pages from non-confidential memory at `source_address` to confidential
+memory at `dest_addr`, then measures and maps the pages at `dest_addr` at the TVM physical
```

Comment:
nit: probably should be consistent with `_address` vs `_addr`

Reply:
Fixed.